changelog

```diff
@@ -1,5 +1,12 @@
 MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki
 
+Version 1.6.0
+-----------
+- Introduces new AadAuthorityAudience enum to support new syntax for specifying cloud + audience
+- Adds new property to IAuthenticationResult: correlation_id
+- Fix for String encoding issue affecting devices with Turkish locale
+- Picks up common@2.0.15
```

Comment:
We also had a couple of bug fixes as well for following:

- https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/1092
- https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/1096

Reply:
Thanks @shahzaibj -- will update

Reply:
Updated in 914c0700c0fcf739b5b652e796091513199d3e16